homeassistant/components/netio/switch.py

```diff
@@ -17,28 +23,27 @@
     EVENT_HOMEASSISTANT_STOP,
     STATE_ON,
 )
-from homeassistant.core import callback
+from homeassistant.core import Event, HomeAssistant, callback
 import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType
+
+from .const import (
+    ATTR_START_DATE,
+    ATTR_TOTAL_CONSUMPTION_KWH,
+    CONF_OUTLETS,
+    DEFAULT_PORT,
+    DEFAULT_USERNAME,
+    MIN_TIME_BETWEEN_SCANS,
+    URL_API_NETIO_EP,
+)
+from .model import Device
 
-_LOGGER = logging.getLogger(__name__)
-
-ATTR_START_DATE = "start_date"
-ATTR_TOTAL_CONSUMPTION_KWH = "total_energy_kwh"
-
-CONF_OUTLETS = "outlets"
-
-DEFAULT_PORT = 1234
-DEFAULT_USERNAME = "admin"
-Device = namedtuple("device", ["netio", "entities"])
-DEVICES = {}
-
-MIN_TIME_BETWEEN_SCANS = timedelta(seconds=10)
-
-REQ_CONF = [CONF_HOST, CONF_OUTLETS]
+_LOGGER: Final = logging.getLogger(__name__)
 
-URL_API_NETIO_EP = "/api/netio/{host}"
+DEVICES: dict[str, Device] = {}
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+PLATFORM_SCHEMA: Final[Schema] = PARENT_PLATFORM_SCHEMA.extend(
```

Comment:
We generally don't alias the parent platform scheme, please revert that.
Was it needed to finalize the schema? If so, why?